next_pr_number.py

```diff
@@ -1,15 +1,21 @@
 import sys
-from github import Github
+
+try:
+    from github3 import GitHub
+except ImportError:
+    raise ImportError('Please conda or pip install github3.py')
 
 if len(sys.argv) == 2:
     repository = sys.argv[1]
 elif len(sys.argv) > 2:
     print("Usage: next_pr_number.py <repository>")
     sys.exit(1)
 else:
-    repository = 'astropy/astropy'
+    repository = ('astropy', 'astropy')
 
-gh = Github()
-repo = gh.get_repo(repository)
-pl = repo.get_issues(sort='created', state='all')
-print("Next PR number: {0}".format(pl.get_page(0)[0].number + 1))
+gh = GitHub()
+repo = gh.repository(*repository)
+pl = repo.issues(sort='created', state='all')
+for issue in repo.issues(sort='created', state='all'):
```

Comment:
Note: `issues()` only returns a generator, so that is why this is implemented as such. I welcome more elegant solution.